src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/ServerCrudTests.ps1

```diff
@@ -107,7 +107,17 @@ function Test-GetServer
 
 		# Test getting all servers in all resource groups
 		$all2 = Get-AzureRmSqlServer
-		Assert-AreEqual 3 $all2.Count
+
+		# It is possible that there were existing servers in the subscription when the test was recorded, so make sure
+		# that the servers that we created are retrieved and ignore the other ones.
```

Comment:
Interestingly that does work. I wasn't aware that trying to access a property on an array would return the property value for each object in the array.